packages/scene-composer/src/components/three-fiber/DataOverlayComponent/DataOverlayDataRow.tsx

```diff
@@ -12,15 +13,16 @@ export const DataOverlayDataRow = ({ rowData, overlayType }: DataOverlayDataRowP
   switch (rowData.rowType) {
     case Component.DataOverlayRowType.Markdown: {
       const row = rowData as Component.DataOverlayMarkdownRow;
-      // TODO: use markdown processor in next change
       return (
-        <p
+        <ReactMarkdown
+          skipHtml
           className={`markdown-row ${
             overlayType === Component.DataOverlaySubType.TextAnnotation ? 'annotation-row' : 'panel-row'
           }`}
+          linkTarget='_blank'
         >
           {row.content}
-        </p>
+        </ReactMarkdown>
```

Comment:
We almost certainly want our own wrapper around this markdown component where we control the styles and structure. The data overlay shouldn't care about our markdown configuration, it should just provide "our markdown input".

Reply:
Please create a wrapper component just for the MarkdownInput component.